src/main/java/com/google/swarm/tokenization/DLPTextToBigQueryStreamingV2PipelineOptions.java

```diff
@@ -141,6 +142,28 @@ public interface DLPTextToBigQueryStreamingV2PipelineOptions
    */
   void setInitialBackoff(int value);
 
+  @Description("Output bucket to write DEID output as csv file")
+  String getOutputBucket();
+
+  void setOutputBucket(String outputBucket);
```

Comment:
Please make the corresponding changes in README.md and update the DAG as well.

Reply:
Ack

Reply:
Will send out a separate pr for the same